CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.3.0] - 2020-11-02
```

Comment:
Feels like this might collide with the other PR?

Reply:
It does collide, but I'm not sure how to handle it (I'm not a developer).

Reply:
just do your pull first ill rebase my changes around yours